susemanager-utils/susemanager-sls/salt/virt/create-vm.sls

```diff
@@ -75,6 +93,8 @@ domain_first_boot_define:
     - require:
       - file: kernel_cached
       - file: initrd_cached
+      - virt_utils: nets-{{ pillar['name'] }}
```

Comment:
IIUC this state you require here might not exist in case `'interfaces' not in pillar` (according to the jinja logic above in the SLS).

Is this correct?

Reply:
oh right!